test/functional/admin/new_document_controller_test.rb

```diff
@@ -0,0 +1,54 @@
+require "test_helper"
```

Comment:
It would be nice if the tests were formatted consistently in terms of blank lines—some have no blank lines, others separate the arrange/act/assert parts. Personally, I find it easier to skim through tests when they are separated into Arrange/Act/Assert sections by [separating each section by blank lines](https://blog.ploeh.dk/2013/06/24/a-heuristic-for-formatting-code-according-to-the-aaa-pattern/#77fd4d083b8541eab04a75783bd46719).

Reply:
Done!

Reply:
This still doesn't seem to be consistent to me? Happy to jump on a quick call if my comment isn't clear.

Reply:
These look good now 👍